docs/library/graphing/charts/piechart.md

```diff
@@ -26,23 +25,28 @@ data01 = [
   },
   {
     "name": "Group B",
-    "value": 300
+    "value": 300,
+    "fill":"#D41009FF"
```

Comment:
Can we use rx.color here instead

Reply:
for all the colors

Reply:
I wanted to use different colors to show we support all of them.
But I can make that change.